src/app/public/modules/progress-indicator/progress-indicator.component.html

```diff
@@ -1,39 +1,15 @@
-<div *ngIf="isHorizontal"
-  class="sky-progress-indicator-display"
+<div
+  class="sky-progress-indicator"
+  [ngClass]="cssClassNames"
 >
-  <div *ngFor="let item of progressItems; let i = index"
-    class="sky-progress-indicator-display-item"
-    [ngClass]="{
-      'sky-progress-indicator-display-item-complete': !item.isActive && item.isComplete
-    }"
+  <div *ngIf="displayModeName === 'horizontal'"
```

Comment:
The `displayModeName` property is public and I didn't want to create yet another public property/method on the component that would make it difficult for us to change later on.

Also, checking for the `enum` in the template would be dangerous because we would be making a check against a numeric value, which could potentially change if more items are added (or moved) in the enum.

For example:

```
// 'Horizontal' might change to 0 or 3 or something else later...
*ngIf="displayMode === 1"
```

Another option would be to create a public method `isHorizontal()` (like Conor had before), but that check is binary, and wouldn't do us much good if we needed to add more than two display modes. Personally, I think the check against the "display mode name" is the easiest to understand from the template's perspective, and doesn't require us to create another public method on the component.

Reply:
Good argument. I'm on the same page.